src/assisted_test_infra/download_logs/__main__.py

```diff
@@ -89,6 +99,11 @@ def handle_arguments():
         type=str,
         default=os.path.join(os.getenv("HOME"), ".kube/config"),
     )
+    parser.add_argument(
+        "--clusters-to-filter-out-file",
+        help="A file containing a list of clusters to ignore, delimeted by a new line",
```

Comment:
Looks like this is expected to be tuple `<cluster_id>,<install_ts>`, isn't it?

Reply:
yes, changed the docs, thanks!